src/main/java/com/jprotractor/NgWebDriver.java

```diff
@@ -11,137 +10,125 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.internal.WrapsDriver;
 
-public class NgWebDriver implements WebDriver, WrapsDriver {
-    private WebDriver driver;
-    private JavascriptExecutor jsExecutor;
-    private String rootElement;
-    @SuppressWarnings("unused")
-    private NgModule[] mockModules;
-    public boolean ignoresync;
```

Comment:
@sergueik hmm, it was not being used, that's why I removed it...

I'll re-add it in another PR